src/Analyzers/CSharp/Analyzers/UseCollectionExpression/AbstractCSharpUseCollectionExpressionDiagnosticAnalyzer.cs

```diff
@@ -26,14 +25,14 @@ internal abstract class AbstractCSharpUseCollectionExpressionDiagnosticAnalyzer
     protected readonly DiagnosticDescriptor UnnecessaryCodeDescriptor;
 
     protected AbstractCSharpUseCollectionExpressionDiagnosticAnalyzer(string diagnosticId, EnforceOnBuild enforceOnBuild)
-        : base(ImmutableDictionary<DiagnosticDescriptor, IOption2>.Empty
-            // Ugly hack.  We need to create a descriptor to pass to our base *and* assign to one of our fields.
-            // The conditional pattern form lets us do that.
```

Comment:
This comment no longer needed because don't need to jump through the hoops to know which item is the descriptor vs UnnecessaryCodeDescriptor. They are in the order this code placed them there.